Removing republishing dids warning

blackgirlbytes opened the pull request:
From chatting back and forth with Gabe and Frank, they advise that we remove this warning about DID DHT because this is currently not the case. At the moment, people don't need to republish DIDs weekly. However, they plan to implement a way to make storing and managing DIDs more sustainable for our company...so that old, unused DIDs aren't just sitting in our system.  Gabe is currently working on implementing new plan. Share the exact and correct guidance with us and we can update our docs to reflect his new guidance.

New plan was agreed on: 
```
person gets a did dht
on our side we're checking to see if the did is active for the last 7 days
if it becomes inactive..they have to republish..if it's not inactive and meets minimal requirements for use..they dont have to republish.
the activity/inactivity is what they refer to as proof of work
```

It sounds like they're still trying to determine what is "activity"

This pull request includes a change in the `how-to-create-did.mdx` file under the `site/docs/web5/build/decentralized-identifiers` directory. The change involves the removal of an important note about the weekly republishing requirement for `did:dht` DIDs. The note stated that DIDs created using the `did:dht` method must be republished weekly to prevent them from becoming unresolvable and that an automatic republishing feature would be integrated into the SDK in the coming weeks.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207249235942948

EbonyLouis reviewed (changes requested):
If this is now the case should we also remove showing how to `republish` ? Since its not needed

blackgirlbytes commented:
> If this is now the case should we also remove showing how to `republish` ? Since its not needed

oh i didnt notice we added that. just removed it!  @EbonyLouis

angiejones commented:
did you mean to include the `connect()` commits in here?

blackgirlbytes commented:
> did you mean to include the `connect()` commits in here?

oh no i didnt! I just did this 2 minutes ago by accident. let me revert @angiejones

blackgirlbytes commented:
Closing this PR!! ughhhh I didn't pay attention and I messed up two branches I had. I'll just redo it.
